### PR TITLE
[RETURN-6047] Update project seed to the new Stack

### PR DESCRIPTION
**Pull request by friscoMad**

### CODE REVIEW

File: project-seed/.buildkite/hooks/post-checkout [hunk 1]
```diff
@@ -1,13 +0,0 @@
-#!/bin/bash
-
-set -euo pipefail
-
-export SHORT_COMMIT_HASH="$(git rev-parse --short HEAD)"
```
Comment (friscoMad) [inline project-seed/.buildkite/hooks/post-checkout:5]: Not needed anymore as we are setting this env vars in our build.sh (I did a test with product-svc and Buildkite builds without this file)

File: project-seed/.buildkite/pipeline.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 steps:
 
   - label: ':coffee: Build, then :docker: Push to ECR'
-    command: common-gradle-scripts/docker-build/build.sh build -Preturnly_env=dev
```
Comment (friscoMad) [inline project-seed/.buildkite/pipeline.yml:4]: Not sure why it was needed

File: project-seed/build.gradle [hunk 3]
```diff
@@ -44,4 +41,5 @@ apply from: "$sharedScripts/spring-boot-starters.gradle"
 apply from: "$sharedScripts/container-app.gradle"
 
 apply from: "$sharedScripts/test-conf.gradle"
-apply from: "$sharedScripts/restdocs.gradle"
+// Remove comment after Asciidoc tests are added
+// apply from: "$sharedScripts/restdocs.gradle"
```
Comment (friscoMad) [inline project-seed/build.gradle:45]: With this change and the default Application class the new project will build out of the box.

File: seed-new-project [hunk 4]
```diff
@@ -31,4 +31,4 @@ find . -name .keep -exec rm '{}' \;
 git init
 git add .
 git commit -m 'Seeding the project'
-git subtree add --prefix common-gradle-scripts git@github.com:returnly/common-gradle-scripts.git master --squash
+git subtree add --prefix common-gradle-scripts git@github.com:returnly/common-gradle-scripts.git master2 --squash
```
Comment (friscoMad) [inline seed-new-project:34]: We will need to change this in the future

File: project-seed/.buildkite/pipeline.yml (hunk 2 above)
Comment (eric239) [reply project-seed/.buildkite/pipeline.yml:4]: I don't recall either; as long as it's not needed effectively (new branches build without it) sure let's remove

**Review by eric239 — APPROVED** [review]

**Review by ruben-returnly — APPROVED** [review]

**Review by tchittick — APPROVED** [review]

**Pull request merged**